Nimble/Matchers/RaisesException.swift

```diff
@@ -10,14 +10,14 @@ import Foundation
 /// nil arguments indicates that the matcher should not attempt to match against
 /// that parameter.
 public func raiseException(
-    named: String? = nil,
+    named named: String? = nil,
```

Comment:
Ah, you're right. I was suggesting keeping the interface the same but renaming the local parameter to "name" instead of "named" – does that make sense? Totally up for a separate PR.